turbine_tank/tank_test.py

```diff
@@ -0,0 +1,134 @@
+# Copyright 2024 Advanced Micro Devices, Inc
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+import pytest
+import tank_util
+from turbine_models.model_builder import HFTransformerBuilder
+
+
+@pytest.mark.parametrize(
+    "model_name,model_type,expected_err,compile_to_vmfb",
+    [
+        ("microsoft/resnet-50", "hf_img_cls", 8e-05, True),
+        ("bert-large-uncased", "hf", 8e-06, True),
```

Comment:
A few questions about these programs with respect to CI integration:
* How much space do each of these take up on disk?
* Can we run correctness tests for these programs using splatted weights?
  * I'd like to commit program .mlir files and splat .irpa files for small/fast (presubmit) tests and then allow large/slow (nightly) tests to download real weights

Reply:
External param mlir files are all mostly under 1 GB. The inline weight versions can get pretty big (up to 30 GB). Not familiar with splatted weights, will sync with you on that.

Reply:
Synced and decided that accuracy still matters here, but when plugging into Scott's setup will make conversion.